fidelisnetwork.json

```diff
@@ -0,0 +1,1082 @@
+                    "data_path": "action_result.data",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.summary",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "example_values": [
+                        1
+                    ],
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "example_values": [
+                        1
+                    ],
+                    "data_type": "numeric"
+                }
+            ],
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "get alert details",
+            "description": "Gets an alert details from Fidelis Network",
+            "type": "investigate",
+            "identifier": "get_alert_details",
+            "read_only": true,
+            "parameters": {
+                "alert_id": {
+                    "description": "Alert ID",
+                    "data_type": "string",
+                    "primary": true,
+                    "required": true,
+                    "order": 0,
+                    "contains": [
+                        "alert id"
+                    ]
+                }
+            },
+            "render": {
+                "type": "table",
+                "width": 12,
+                "height": 5,
+                "title": "Get Alert Details"
+            },
+            "output": [
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "example_values": [
+                        "success",
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.alertId",
+                    "example_values": [
+                        4086
+                    ],
+                    "column_name": "Alert ID",
+                    "column_order": 0,
+                    "contains": [
+                        "alert id"
+                    ],
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "action_result.data.*.severity",
+                    "example_values": [
+                        "Medium"
+                    ],
+                    "column_name": "SEVERITY",
+                    "column_order": 1,
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.time",
+                    "example_values": [
+                        "Medium"
+                    ],
```

Comment:
this value should be timestamp, right?

Reply:
Yes, This should be a timestamp value. I have updated it.